testing/kuttl/e2e/root-cert-ownership/01-check-owners.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Get a list of the current owners of the root ca cert secret and verify that
+  # both owners are listed.
+  - script: |
+      CURRENT_OWNERS=$(kubectl --namespace="${NAMESPACE}" get secret \
+      pgo-root-cacert -o jsonpath='{.metadata.ownerReferences[*].name}')
+      echo "$CURRENT_OWNERS"
```

Comment:
Don't need this in the test, do we? Hmmm, you know, now that I'm thinking about it, I've been removing all these echoes from my tests, but if they're useful for debugging maybe they're also useful in testing?

(I'd still remove it for now since that's mostly our style, right? But might be worth a chat.)

Reply:
They are useful for debugging, but I think you're correct on the general style we've been using thus far. I'll remove them for now, we can always revisit this later and approach things more holistically.